.yamato/standalone-build-test.yml

```diff
@@ -15,10 +15,13 @@ test_mac_standalone_{{ editor.version }}:
     - pip install pyyaml
     - python -u -m ml-agents.tests.yamato.standalone_build_tests
   triggers:
-    pull_requests:
-      - targets:
-          only:
-            - "master"
-            - "/release-.*/"
-            - "/hotfix-.*/"
+    changes:
+      only:
+        - "com.unity.ml-agents/**"
+        - "Project/**"
+        - ".yamato/standalone-build-test.yml"
+      exept:
```

Comment:
exept or except?

Reply:
Looks like `except` according to https://internaldocs.hq.unity3d.com/yamato/yaml/triggers/#file-changes

Reply:
Whoops, typo